### PR TITLE
Correct the sha in http server.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Require https://github.com/vesoft-inc/nebula-common/pull/415

### CODE REVIEW

**Comment by critical27** [comment]
Do we need to remove `src/version/Version.cpp.in`?

**Comment by Shylock-Hg** [comment]
> Do we need to remove `src/version/Version.cpp.in`?

Good catch, I think it's duplicate now.

File: src/version/test/StatusHandlerTest.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
```
Comment (bright-starry-sky) [inline src/version/test/StatusHandlerTest.cpp:1]: 2121

Comment (Shylock-Hg) [reply src/version/test/StatusHandlerTest.cpp]: Fixed.

**Review by critical27 — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
LGTM. please check the cmake error.

**Comment by Shylock-Hg** [comment]
> LGTM. please check the cmake error.

This pr depends on another pr of common that I mentioned above

**Review by critical27 — APPROVED** [review]

**Pull request merged**